general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -335,9 +336,9 @@ def compile(self, json_dict: dict[str, str]) -> dict[str, str]:
 
     def submit_qubo(
         self,
-        qubo: dict[tuple[()] | tuple[str | int] | tuple[str | int, str | int], int | float],
+        qubo: Mapping[tuple[TQuboKey, ...], float],
```

Comment:
One question I have is whether we should have some client-side validation of `qubo` to check that the qubo keys have a max length of 2, i.e., only quadratic terms

Reply:
good idea, just added a validator